libs/components/flyout/src/lib/modules/flyout/flyout.component.ts

```diff
@@ -58,6 +59,11 @@ let nextId = 0;
     SkyFlyoutAdapterService,
     SkyFlyoutMediaQueryService,
     { provide: SkyMediaQueryService, useExisting: SkyFlyoutMediaQueryService },
+    {
+      provide: SkyStackingContextService,
+      // Match z-index set in libs/components/theme/src/lib/styles/_public-api/_compat/_variables.scss
+      useValue: new SkyStackingContextService({ zIndex: 1001 }),
```

Comment:
Would it be possible to refactor the styles so that the flyout component uses the z-index you're defining here, instead? The fact that you can update the `zIndex` value here and it has no effect on the component is not ideal (two places the z-index needs to be updated/tracked).

Also, I would make this value an observable and a property of the component in case the flyout needs to change the z-index for any reason.

Reply:
Refactored and added comments to the scss variables file.